src/pages/cli/usage/command-hooks.mdx

```diff
@@ -0,0 +1,250 @@
+export const meta = {
+  title: `Command Hooks`,
+  description: `Execute custom scripts before, during, and after Amplify CLI commands (“amplify push”, “amplify api gql-compile”, and more). This allows you to extend Amplify’s best-practice defaults to meet your organization’s specific security guidelines and operational requirements.`,
+};
+
+Use Command Hooks to execute custom scripts before, during, and after Amplify CLI commands (“amplify push”, “amplify api gql-compile”, and more). This allows you to extend Amplify’s best-practice defaults to meet your organization’s specific security guidelines and operational requirements. 
+
+## Adding a command hook
+Place your custom scripts in the `amplify/hooks` directory and set the script file name to the desired command with a pre or post designation. For example `post-add-function.js` will execute the script after `amplify add function`. For more information about the script naming convention, see [How to name command hook scripts](#how-to-name-command-hook-scripts). 
```

Comment:
Can we also mention , hooks folder will get created when running amplify init with sample hooks

Reply:
Already mentioned below.